.rubocop.yml

```diff
@@ -451,6 +460,8 @@ Capybara/SpecificFinders: # new in 2.13
   Enabled: true
 Capybara/SpecificMatcher: # new in 2.12
   Enabled: true
+  Exclude:
+    - spec/components/elements/forms/submit_component_spec.rb # we want to explicitly test for an HTML input with type="submit"
```

Comment:
Since we intend to skip this Rubocop violation, move it from an inline skip to a configured skip with a rationale.